contracts/StakingRewardsV2.sol

```diff
@@ -343,10 +352,25 @@ contract StakingRewardsV2 is
             // emit reward claimed event and index account
             emit RewardPaid(_account, reward);
 
-            // transfer token from this contract to the rewardEscrow
-            // and create a vesting entry at the _to address
-            kwenta.transfer(address(rewardEscrow), reward);
-            rewardEscrow.appendVestingEntry(_to, reward);
+            // transfer token from this contract to the account
+            // as newly issued rewards from inflation are now issued as non-escrowed
+            kwenta.transfer(_to, reward);
+        }
+    }
+
+    function _getRewardCompounding(address _account)
```

Comment:
This is called when compounding, basically it is doing the same as `_getReward` but without transfering because the rewards are staked right after this (can be staked immediatly after calling this (see `_compound()`).

Added natspec to specify this.

Reply:
Hmm, is there a difference to this vs. the code in _getReward other than transferring? I see duplicate code. I was thinking that this reward calculating logic would be shared between the two and getReward only has the distinction of transferring to the user.

ie.
```solidity
getReward() {
    (kwenta, usdc) = _calculateReward()
    transfer kwenta
    transfer USDC
}

compound() {
    (kwenta, usdc) = _calculateReward()
    stake kwenta
    transfer USDC
}

_calculateReward() {
...
}
```

TLDR; we should be practicing [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) here to minimize room for error.

Reply:
Refactored with a `_processReward` so that there is no duplicate logic